cypresse2e/cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
```

Comment:
Do we need this example file?

Reply:
Hi @idjevm 
This file will run before every single spec file. We can here put reusable behaviour such as Custom Commands or global overrides -> available to all of spec files.